docs/en/developers/references/traits/route.md

```diff
@@ -1,5 +1,27 @@
 # Route
 
+## Description
+
+`Route` is used to configure external access to your service.
+
+## Specification
+
+List of all available properties for a `Route` trait.
+
+```yaml
+name: my-app-name
+
+services:
+  my-service-name:
+    ...
+    route:
+      domain: example.com
+      issuer: tls
+      rules:
+        - path: /testapp
+          rewriteTarget: /
+```
+
```

Comment:
Great, that's the nice reference for developers to write appfile at it's said in #577 :)